lib/core/src/text.rs

```diff
@@ -158,14 +194,14 @@ impl<'a> TextComponentBuilder<'a> {
         let tex_image_result =
             gl_ctx.tex_image_2d_with_i32_and_i32_and_i32_and_format_and_type_and_opt_u8_array
             ( target
-            , tex_level
-            , internal_fmt
-            , width
-            , height
-            , border
-            , format
-            , tex_type
-            , data
+                , tex_level
+                , internal_fmt
+                , width
+                , height
+                , border
+                , format
+                , tex_type
+                , data
```

Comment:
Again minor thing but I assume this change (and the one above) was accidental

Reply:
Yeah, that's Intellij work. If you know how to tell him "be less eager in autoformatting", give me a clue.